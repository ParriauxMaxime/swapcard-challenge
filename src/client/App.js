import React from 'react';
import { Route } from 'react-router-dom';
import { About } from "./Component/About";
import { ConnectedHome } from "./Component/Home";

const App = () => (
  <div>
    <Route exact path="/" component={ConnectedHome} />
    <Route exact path="/about" component={About} />
  </div>
);

export default App;
