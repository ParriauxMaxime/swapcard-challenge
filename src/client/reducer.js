import { combineReducers } from 'redux';
import { routerReducer } from 'react-router-redux'
import search from './reducers/search';
import spotify from './reducers/spotify'
import artist from './reducers/artist'

export default combineReducers({
  search,
  spotify,
  artist,
  router: routerReducer
});
