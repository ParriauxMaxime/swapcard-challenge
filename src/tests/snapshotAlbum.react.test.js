const { createElement: e } = require('react');
const { Provider } = require('react-redux');
const { createStore } = require('redux');
const { StaticRouter } = require('react-router-dom');
const renderer = require('react-test-renderer');
const MuiThemeProvider = require('material-ui/styles').MuiThemeProvider;

const theme = require('../client/theme').theme;
const { addAlbums, selectAlbum, addTracks } = require('../client/Api/action');

import App from '../client/App';
import reducer from '../client/reducer';

const mockup = require('./mockup.json').album


it('renders correctly', () => {
  const store = createStore(reducer);
  store.dispatch(selectAlbum(mockup.id))
  store.dispatch(addAlbums([mockup]))
  store.dispatch(addTracks(mockup.tracks.items))


  //Damso - Ipséité
  const url = '/album/7ovdjtmV0Bkm4Xb4pqtrZt'
  const context = {}
  const elem = e(Provider, { store },
                  e(MuiThemeProvider, {theme}, 
                      e(StaticRouter, { 
                          location: url,
                          context
                        },
                        e(App)
                )
              )
            );
  const tree = renderer
    .create(elem)
    .toJSON();
  expect(tree).toMatchSnapshot();
});