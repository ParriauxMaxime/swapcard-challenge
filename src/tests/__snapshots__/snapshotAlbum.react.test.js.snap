// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexDirection": "column",
      "maxHeight": "100%",
    }
  }
>
  <header
    className="MuiPaper-root-9 MuiPaper-elevation4-15 MuiAppBar-root-1 MuiAppBar-positionStatic-5 MuiAppBar-colorPrimary-7"
  >
    <div
      className="MuiToolbar-root-36 MuiToolbar-gutters-37"
    >
      <h1>
        Spotify fetcher
      </h1>
    </div>
  </header>
  <div
    className="MuiGrid-container-44 AlbumView-root-38"
  >
    <div
      className="MuiGrid-item-45 MuiGrid-grid-xs-12-81 MuiGrid-grid-md-6-101 AlbumView-left-39"
    >
      <div
        className="MuiPaper-root-9 MuiPaper-elevation2-13 MuiPaper-rounded-10 AlbumView-paperInfo-42"
      >
        <button
          className="MuiButtonBase-root-149 MuiButton-root-134 AlbumView-goBack-40"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex="0"
          type="button"
        >
          <span
            className="MuiButton-label-135"
          >
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root-151"
              color={undefined}
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z"
              />
            </svg>
          </span>
          <span
            className="MuiTouchRipple-root-157"
          />
        </button>
        <img
          alt="Ipséité album"
          className="AlbumView-albumImg-43"
          src="https://i.scdn.co/image/2b0545ca873f01d21841dc3f192b7317be9bf3c3"
        />
         
        <h1
          className="MuiTypography-root-164 MuiTypography-display1-168"
        >
          Ipséité
        </h1>
         
        <div>
          <button
            className="MuiButtonBase-root-149 MuiButton-root-134"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex="0"
            type="button"
          >
            <span
              className="MuiButton-label-135"
            >
              <span
                className="MuiTypography-root-164 MuiTypography-caption-174"
              >
                Damso
                
              </span>
            </span>
            <span
              className="MuiTouchRipple-root-157"
            />
          </button>
        </div>
        <span
          className="MuiTypography-root-164 MuiTypography-caption-174"
        >
          2017 -
          14 tracks
        </span>
         
        <span
          className="MuiTypography-root-164 MuiTypography-caption-174"
        >
          AZ
        </span>
      </div>
    </div>
    <div
      className="MuiGrid-item-45 MuiGrid-grid-xs-12-81 MuiGrid-grid-md-6-101 AlbumView-right-41"
    >
      <ul
        className="MuiList-root-189 TracksList-root-188"
      >
        <li
          className="MuiListItem-root-193 MuiListItem-default-196 MuiListItem-gutters-200"
          disabled={false}
        >
          <div
            className="MuiListItemText-root-203"
          >
            <h3
              className="MuiTypography-root-164 MuiTypography-subheading-171 MuiListItemText-primary-206"
            >
              Disc 
              1
            </h3>
          </div>
        </li>
        <li
          className="MuiListItem-root-193 MuiListItem-default-196 MuiListItem-gutters-200"
          disabled={false}
        >
          <ul
            className="MuiList-root-189 TracksList-root-188"
          >
            <li
              className="MuiListItem-root-193 MuiListItem-default-196 MuiListItem-gutters-200"
              disabled={false}
            >
              <div
                className="MuiGrid-container-44"
              >
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <h2
                    className="MuiTypography-root-164 MuiTypography-title-170"
                  >
                    #
                  </h2>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-8-77"
                >
                  <h2
                    className="MuiTypography-root-164 MuiTypography-title-170"
                  >
                    Name
                  </h2>
                </div>
              </div>
            </li>
            <hr
              className="MuiDivider-root-209"
            />
            <li
              className="MuiListItem-root-193 MuiListItem-default-196 MuiListItem-gutters-200"
              disabled={false}
            >
              <div
                className="MuiGrid-container-44"
              >
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    1
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-8-77"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    Α. Nwaar Is The New Black
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-2-71"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    02
                     : 
                    04
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <a
                    href="https://open.spotify.com/track/34iEMu8pGMHsZtNqAoyVZD"
                    style={
                      Object {
                        "color": "black",
                      }
                    }
                  >
                    <svg
                      aria-hidden="true"
                      className="MuiSvgIcon-root-151"
                      color={undefined}
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M6 10c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm12 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm-6 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                      />
                    </svg>
                  </a>
                </div>
              </div>
            </li>
            <hr
              className="MuiDivider-root-209"
            />
            <li
              className="MuiListItem-root-193 MuiListItem-default-196 MuiListItem-gutters-200"
              disabled={false}
            >
              <div
                className="MuiGrid-container-44"
              >
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    2
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-8-77"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    B. #QuedusaalVie
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-2-71"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    04
                     : 
                    05
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <a
                    href="https://open.spotify.com/track/7xNP9UhpmWFFp2jFuYGM3m"
                    style={
                      Object {
                        "color": "black",
                      }
                    }
                  >
                    <svg
                      aria-hidden="true"
                      className="MuiSvgIcon-root-151"
                      color={undefined}
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M6 10c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm12 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm-6 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                      />
                    </svg>
                  </a>
                </div>
              </div>
            </li>
            <hr
              className="MuiDivider-root-209"
            />
            <li
              className="MuiListItem-root-193 MuiListItem-default-196 MuiListItem-gutters-200"
              disabled={false}
            >
              <div
                className="MuiGrid-container-44"
              >
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    3
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-8-77"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    Γ. Mosaïque solitaire
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-2-71"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    05
                     : 
                    06
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <a
                    href="https://open.spotify.com/track/7wFK2ReVQpq80IAGs6eI1l"
                    style={
                      Object {
                        "color": "black",
                      }
                    }
                  >
                    <svg
                      aria-hidden="true"
                      className="MuiSvgIcon-root-151"
                      color={undefined}
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M6 10c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm12 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm-6 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                      />
                    </svg>
                  </a>
                </div>
              </div>
            </li>
            <hr
              className="MuiDivider-root-209"
            />
            <li
              className="MuiListItem-root-193 MuiListItem-default-196 MuiListItem-gutters-200"
              disabled={false}
            >
              <div
                className="MuiGrid-container-44"
              >
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    4
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-8-77"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    Δ. Dieu ne ment jamais
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-2-71"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    03
                     : 
                    19
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <a
                    href="https://open.spotify.com/track/6YWjskKykdPsBuiTBOg1VK"
                    style={
                      Object {
                        "color": "black",
                      }
                    }
                  >
                    <svg
                      aria-hidden="true"
                      className="MuiSvgIcon-root-151"
                      color={undefined}
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M6 10c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm12 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm-6 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                      />
                    </svg>
                  </a>
                </div>
              </div>
            </li>
            <hr
              className="MuiDivider-root-209"
            />
            <li
              className="MuiListItem-root-193 MuiListItem-default-196 MuiListItem-gutters-200"
              disabled={false}
            >
              <div
                className="MuiGrid-container-44"
              >
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    5
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-8-77"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    Ε. Signaler
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-2-71"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    03
                     : 
                    22
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <a
                    href="https://open.spotify.com/track/0z0izoGFPCoPJKFi41azyb"
                    style={
                      Object {
                        "color": "black",
                      }
                    }
                  >
                    <svg
                      aria-hidden="true"
                      className="MuiSvgIcon-root-151"
                      color={undefined}
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M6 10c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm12 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm-6 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                      />
                    </svg>
                  </a>
                </div>
              </div>
            </li>
            <hr
              className="MuiDivider-root-209"
            />
            <li
              className="MuiListItem-root-193 MuiListItem-default-196 MuiListItem-gutters-200"
              disabled={false}
            >
              <div
                className="MuiGrid-container-44"
              >
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    6
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-8-77"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    Ζ. Kietu
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-2-71"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    03
                     : 
                    52
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <a
                    href="https://open.spotify.com/track/05w1S3f4G9ulCktPrygay4"
                    style={
                      Object {
                        "color": "black",
                      }
                    }
                  >
                    <svg
                      aria-hidden="true"
                      className="MuiSvgIcon-root-151"
                      color={undefined}
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M6 10c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm12 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm-6 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                      />
                    </svg>
                  </a>
                </div>
              </div>
            </li>
            <hr
              className="MuiDivider-root-209"
            />
            <li
              className="MuiListItem-root-193 MuiListItem-default-196 MuiListItem-gutters-200"
              disabled={false}
            >
              <div
                className="MuiGrid-container-44"
              >
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    7
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-8-77"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    Η. Gova
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-2-71"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    03
                     : 
                    17
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <a
                    href="https://open.spotify.com/track/4LDCalloaDBsmhBk3BYP4a"
                    style={
                      Object {
                        "color": "black",
                      }
                    }
                  >
                    <svg
                      aria-hidden="true"
                      className="MuiSvgIcon-root-151"
                      color={undefined}
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M6 10c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm12 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm-6 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                      />
                    </svg>
                  </a>
                </div>
              </div>
            </li>
            <hr
              className="MuiDivider-root-209"
            />
            <li
              className="MuiListItem-root-193 MuiListItem-default-196 MuiListItem-gutters-200"
              disabled={false}
            >
              <div
                className="MuiGrid-container-44"
              >
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    8
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-8-77"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    Θ. Macarena
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-2-71"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    03
                     : 
                    27
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <a
                    href="https://open.spotify.com/track/2ewjMyCbNv2X1dB2qIDCwD"
                    style={
                      Object {
                        "color": "black",
                      }
                    }
                  >
                    <svg
                      aria-hidden="true"
                      className="MuiSvgIcon-root-151"
                      color={undefined}
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M6 10c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm12 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm-6 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                      />
                    </svg>
                  </a>
                </div>
              </div>
            </li>
            <hr
              className="MuiDivider-root-209"
            />
            <li
              className="MuiListItem-root-193 MuiListItem-default-196 MuiListItem-gutters-200"
              disabled={false}
            >
              <div
                className="MuiGrid-container-44"
              >
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    9
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-8-77"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    Ι. Peur d'être père
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-2-71"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    03
                     : 
                    25
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <a
                    href="https://open.spotify.com/track/10CJ0c1v7m9PiPVyieJ0hG"
                    style={
                      Object {
                        "color": "black",
                      }
                    }
                  >
                    <svg
                      aria-hidden="true"
                      className="MuiSvgIcon-root-151"
                      color={undefined}
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M6 10c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm12 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm-6 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                      />
                    </svg>
                  </a>
                </div>
              </div>
            </li>
            <hr
              className="MuiDivider-root-209"
            />
            <li
              className="MuiListItem-root-193 MuiListItem-default-196 MuiListItem-gutters-200"
              disabled={false}
            >
              <div
                className="MuiGrid-container-44"
              >
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    10
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-8-77"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    Κ. Kin la belle
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-2-71"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    03
                     : 
                    21
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <a
                    href="https://open.spotify.com/track/6YuSiFqJP4MUo88RtUAfAX"
                    style={
                      Object {
                        "color": "black",
                      }
                    }
                  >
                    <svg
                      aria-hidden="true"
                      className="MuiSvgIcon-root-151"
                      color={undefined}
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M6 10c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm12 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm-6 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                      />
                    </svg>
                  </a>
                </div>
              </div>
            </li>
            <hr
              className="MuiDivider-root-209"
            />
            <li
              className="MuiListItem-root-193 MuiListItem-default-196 MuiListItem-gutters-200"
              disabled={false}
            >
              <div
                className="MuiGrid-container-44"
              >
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    11
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-8-77"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    Λ. Lové
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-2-71"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    03
                     : 
                    30
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <a
                    href="https://open.spotify.com/track/0YHVHL8qSEH5BqZPd1njUz"
                    style={
                      Object {
                        "color": "black",
                      }
                    }
                  >
                    <svg
                      aria-hidden="true"
                      className="MuiSvgIcon-root-151"
                      color={undefined}
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M6 10c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm12 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm-6 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                      />
                    </svg>
                  </a>
                </div>
              </div>
            </li>
            <hr
              className="MuiDivider-root-209"
            />
            <li
              className="MuiListItem-root-193 MuiListItem-default-196 MuiListItem-gutters-200"
              disabled={false}
            >
              <div
                className="MuiGrid-container-44"
              >
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    12
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-8-77"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    Μ. Noob Saibot
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-2-71"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    03
                     : 
                    22
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <a
                    href="https://open.spotify.com/track/505Rb1eFTYTSTwFeNQI2at"
                    style={
                      Object {
                        "color": "black",
                      }
                    }
                  >
                    <svg
                      aria-hidden="true"
                      className="MuiSvgIcon-root-151"
                      color={undefined}
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M6 10c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm12 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm-6 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                      />
                    </svg>
                  </a>
                </div>
              </div>
            </li>
            <hr
              className="MuiDivider-root-209"
            />
            <li
              className="MuiListItem-root-193 MuiListItem-default-196 MuiListItem-gutters-200"
              disabled={false}
            >
              <div
                className="MuiGrid-container-44"
              >
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    13
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-8-77"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    N. J Respect R
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-2-71"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    03
                     : 
                    37
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <a
                    href="https://open.spotify.com/track/539zyjwDE87Cjp1mzRN8H7"
                    style={
                      Object {
                        "color": "black",
                      }
                    }
                  >
                    <svg
                      aria-hidden="true"
                      className="MuiSvgIcon-root-151"
                      color={undefined}
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M6 10c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm12 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm-6 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                      />
                    </svg>
                  </a>
                </div>
              </div>
            </li>
            <hr
              className="MuiDivider-root-209"
            />
            <li
              className="MuiListItem-root-193 MuiListItem-default-196 MuiListItem-gutters-200"
              disabled={false}
            >
              <div
                className="MuiGrid-container-44"
              >
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    14
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-8-77"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    Ξ. Une âme pour deux
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-2-71"
                >
                  <h3
                    className="MuiTypography-root-164 MuiTypography-subheading-171"
                  >
                    03
                     : 
                    60
                  </h3>
                </div>
                <div
                  className="MuiGrid-item-45 MuiGrid-grid-xs-1-70"
                >
                  <a
                    href="https://open.spotify.com/track/395h3MnAkHiBl1bDsoAKTd"
                    style={
                      Object {
                        "color": "black",
                      }
                    }
                  >
                    <svg
                      aria-hidden="true"
                      className="MuiSvgIcon-root-151"
                      color={undefined}
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M6 10c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm12 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm-6 0c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
                      />
                    </svg>
                  </a>
                </div>
              </div>
            </li>
            <hr
              className="MuiDivider-root-209"
            />
          </ul>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
